src/components/enterprise-user-subsidy/tests/UserSubsidy.test.jsx

```diff
@@ -57,44 +75,44 @@ const UserSubsidyWithAppContext = ({ contextValue = {}, children }) => (
     </UserSubsidy>
   </AppContext.Provider>
 );
-
-const HasAccessConsumer = () => {
-  const { hasAccessToPortal } = useContext(UserSubsidyContext);
-  return <div>Has access: {hasAccessToPortal ? 'true' : 'false'} </div>;
-};
+/* eslint-enable react/prop-types */
 
 const SubscriptionLicenseConsumer = () => {
   const { subscriptionLicense } = useContext(UserSubsidyContext);
-  return <div>License status: {subscriptionLicense.status}</div>;
+  return <div>License status: {subscriptionLicense?.status || 'none'}</div>;
 };
 
 const OffersConsumer = () => {
   const { offers } = useContext(UserSubsidyContext);
-  return <div>Offers count: {offers.offersCount}</div>;
+  return <div>Offers count: {offers?.offersCount || 'none'}</div>;
```

Comment:
Not sure how important this is to your test but just wanted to call out when `offers?.offersCount` is 0 none will be displayed here

Reply:
Yeah, that crossed my mind too and I think it's fine if for the purpose of these tests it comes back as `none`. Good callout, though!